ogenerrors/handler.go

```diff
@@ -49,5 +47,7 @@ func DefaultErrorHandler(ctx context.Context, w http.ResponseWriter, r *http.Req
 	e.StrEscape(err.Error())
 	e.ObjEnd()
 
-	_, _ = w.Write(e.Bytes())
+	if _, writeErr := w.Write(e.Bytes()); writeErr != nil {
+		http.Error(w, writeErr.Error(), http.StatusInternalServerError)
+	}
```

Comment:
1. `http.Error` will call `WriteHeader` again. It will lead to `superfluous response.WriteHeader call` error.
2. Sending IO error to client does not make much sense for me.

Reply:
@tdakkota hey!
Thanks!
I close this PR.